GovUkHtmlHelperExtensions.cs

```diff
@@ -295,5 +296,26 @@ public static class GovUkHtmlHelperExtensions
                 classes,
                 textInputAppendix);
         }
+
+        public static IHtmlContent GovUkFileUploadFor<TModel>(
+            this IHtmlHelper<TModel> htmlHelper,
+            Expression<Func<TModel, IFormFile>> propertyExpression,
+            LabelViewModel labelOptions = null,
+            HintViewModel hintOptions = null,
+            FormGroupViewModel formGroupOptions = null,
+            string classes = null,
+            TextInputAppendixViewModel textInputAppendix = null)
+            where TModel : class
+        {
+            return TextInputHtmlGenerator.GenerateHtml(

```

Comment:
Shouldn't this use something like FileInputHtmlGenerator? (Which I can't see in this review?)